l10n/css.json

```diff
@@ -451,6 +451,12 @@
     "ja": "ブロックコンテナ",
     "ru": "блочные контейнеры"
   },
+  "blockContainersFlexContainersGridContainers": {
+    "en-US": "block-containers, flex containers, and grid containers"
```

Comment:
Nit: this should have a capital initial:

"Block-containers, flex containers, and grid containers"

Reply:
This is hard to tell of your own. At first glance it looks like half of them are capitalized and the other half isn't. It guess it maybe depends on where it's used?

Reply:
You're right, this is just inconsistent for no good reason that I can see. It is used in the "Applies to" piece of the blue box, in places like https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius#Specifications. As far as I know this is the only place it is used in MDN, although it might be used in other applications.

In the MDN case, it seems to me that capitalization would be better, but most cases actually seem to not capitalize.

I think it's best to just do what you like here, and we will fix it up later :).

Reply:
Caps it is then 😄